docs/user/aws/customization.md

```diff
@@ -17,14 +17,16 @@ An example `install-config.yaml` is shown below. This configuration has been mod
 ```yaml
 apiVersion: v1beta1
```

Comment:
should this be `beta3`?

Reply:
@akostadinov yeah, #1285 has the update.